components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/list-service-providers.jsp

```diff
@@ -1,145 +1,188 @@
-											key="field.service.provider.id" /></th>
-									<th class="leftCol-big"><fmt:message
-											key="application.list.application.desc" /></th>
-									<th style="width: 30%"><fmt:message
-											key="application.list.application.action" /></th>
-								</tr>
-							</thead>
-							<%
-								if(applications != null && applications.length > 0) {
-							%>
-							<tbody>
-							<%
-								for(ApplicationBasicInfo app : applications) {
-							%>
-								<tr>
-									<td><%=app.getApplicationName()%></td>
-									<td><%=app.getDescription() != null ? app.getDescription() : ""%></td>
-									<td style="width: 100px; white-space: nowrap;"><a
-										title="Edit Service Providers" href="load-service-provider.jsp?spName=<%=app.getApplicationName()%>" class="icon-link"
-										style="background-image: url(../admin/images/edit.gif)">Edit</a>
-
-										<a title="Remove Service Providers" onclick="removeItem('<%=app.getApplicationName()%>');return false;" href="#" class="icon-link"
-										style="background-image: url(../admin/images/delete.gif)">Delete
-									</a></td>
-								</tr>
-								<% } %>
-							</tbody>
-							<% } else { %>
-							<tbody>
-							<tr><td colspan="3"><i>No Service Providers registered</i></td></tr>
-							</tbody>
-							<% } %>
-						</table>
-						</td>
-					</tr>
-				</tbody>
-			</table>
-			<carbon:paginator pageNumber="0"
-                                      numberOfPages="1"
-                                      page="list-service-providers.jsp"
-                                      pageNumberParameterName="pageNumber"
-                                      parameters="10"
-                                      resourceBundle="org.wso2.carbon.identity.application.mgt.ui.i18n.Resources"
-                                      prevKey="prev" nextKey="next"/>
-		</div>
-	</div>
-</fmt:bundle>
+            <table style="width: 100%" class="styledLeft">
+                <tbody>
+                <tr>
+                    <td style="border:none !important">
+                        <!-- SAML SSO -->
+                        <table class="styledLeft" width="100%" id="ServiceProviders">
+                            <thead>
+                            <tr style="white-space: nowrap">
+                                <th class="leftCol-med"><fmt:message
+                                        key="field.service.provider.id"/></th>
+                                <th class="leftCol-big"><fmt:message
+                                        key="application.list.application.desc"/></th>
+                                <th style="width: 30%"><fmt:message
+                                        key="application.list.application.action"/></th>
+                            </tr>
+                            </thead>
+                            <%
+                                if (applications != null && applications.length > 0) {
```

Comment:
Remove the if condition, now "applicationsToDisplay" array is used instead "applications" array